idea/src/org/jetbrains/kotlin/idea/configuration/KotlinLanguageConfiguration.kt

```diff
@@ -159,7 +159,8 @@ class KotlinUpdatesSettingsConfigurable : SearchableConfigurable, Configurable.N
 
                 is PluginUpdateStatus.Unverified -> {
                     val version = pluginUpdateStatus.updateStatus.pluginDescriptor.version
-                    form.setUpdateStatus("A new version $version is found, but not yet verified by Android Studio: ${pluginUpdateStatus.message}", false)
+                    val message = "<html>A new version $version is found, but unverified:<br/> ${pluginUpdateStatus.message}</html>"
```

Comment:
I this message should have a common and reasoning part as a separate sentence. It might not flexible enough to have extensions contributing to the message, as you must change full message thinking about all possible contributors, and you have to write extensions with the full message in mind.

Also, it's now unclear who is responsible for the unverified result.

I have prepared c31a6af98c4062723b7a61eaa3e94d5b623012d2 commit with adressesing these concerns. Can you please check if such solution suitable for you?

Reply:
The separation of reasoning makes sense. Thanks!